services/fuzzilli/fuzzilli.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -e
+set -x
+set -o pipefail
+
+function retry () {
+  op="$(mktemp)"
+  for _ in {1..9}; do
+    if "$@" >"$op"; then
+      cat "$op"
+      rm "$op"
+      return
+    fi
+    sleep 30
+  done
+  rm "$op"
+  "$@"
+}
+
+function tc-get-secret () {
```

Comment:
This is similarly defined in `common.sh` as `get-tc-secret`. eg. `get-tc-secret deploy-fuzzilli $HOME/.ssh/id_rsa.fuzzilli` (it does the json decoding and `chmod 0600` internally).